jobs/build/oc_sync/publish-clients-from-payload.sh

```diff
@@ -1,55 +1,114 @@
 #!/bin/bash
-set -eux
-export MOBY_DISABLE_PIGZ=true
-WORKSPACE=$1
-STREAM=$2
-MIRROR=$3
-OC_MIRROR_DIR="/srv/pub/openshift-v4/clients/$MIRROR/"
-
-SSH_OPTS="-l jenkins_aos_cd_bot -o StrictHostKeychecking=no use-mirror-upload.ops.rhcloud.com"
-
-# get latest release from release-controller API
-wget https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/${STREAM}/latest -O latest
-
-#extract pull_spec
-PULL_SPEC=`jq -r '.pullSpec' latest`
-if [ "$MIRROR" = "ocp-dev-preview" ]; then
-  # point at the published pre-release that will stay around -- registry.svc.ci gets GCed
-  PULL_SPEC="${PULL_SPEC/registry.svc.ci.openshift.org\/ocp\/release/quay.io/openshift-release-dev/ocp-release-nightly}"
-fi
 
-#extract name
-VERSION=`jq -r '.name' latest`
+main() {
+  set -euxo pipefail
```

Comment:
This seems like it ought to stay global. It has more or less the same effect being at the beginning but... it just looks wrong and make me think.